### DISCUSSION TITLE
Slideout / Element Editor Missing Enable Lightswitch

### DISCUSSION THREAD

**Discussion by RitterKnightCreative** [craftcms/cms#8515] [Ideas] [answered]
### Description
We use a lot entry fields to relate content together. 

The editor/modal editing feature introduced in 2.3 (I think?) lets you edit related entries without leaving the entry you're currently on. That's super handy except when you need to turn on the entry.

Craft also allows you to select disabled entries.  Oftentimes it becomes necessary to edit those entries and then enable them.

However, the enable/disable lightswitch doesn't show up in the popover editor where it would be super convenient to do so.

### Steps to reproduce

1. Invoke the editor modal
2. Can't enable entry from modal.

**Comment by RitterKnightCreative** [comment]
Now that slideouts are a thing in in 3.7 and since they show meta data, there should now be enough room to add the Lightswitch to enable and disable entries from the slider which would easily solve this.

**Reply by brandonkelly** [reply]
Needs to wait until 4.0, because elements don’t currently have a way to communicate whether they should get per-site statuses or not.

**Comment by brandonkelly** [comment, marked as answer]
This was added in Craft 4 🎉
